tests/integration/components/frost-fixed-table-test.js

```diff
@@ -506,6 +509,21 @@ describe(test.label, function () {
         })
       })
     })
+
+    describe('clearing the selection', function () {
+      beforeEach(function () {
+        rowBodySingleSelect('myTable-left', 0)
+        return wait().then(() => {
+          _assertRowsSelected(0)
+          $hook('myTable-header-left-selectionCell').click()
```

Comment:
Not being knowledgeable about `frost-table` I don't know that you absolutely need to protect all this with `wait` unless you have async stuff running of the select events.